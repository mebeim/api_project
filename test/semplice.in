create_dir  /foo
create /foo/bar
create_dir /foo/baz
create /foo/baz/qux
write  /foo/bar "test"
read   /foo/bar
read   /foo/baz/qux
read   /foo/baz/quux
create /foo/bar
create_dir /dir
create_dir /bar
create /dir/bar
find bar
delete /foo/bar
find wat
find foo
read   /foo/bar
create /foo/bar
read   /foo/bar
delete_r /foo
exit

