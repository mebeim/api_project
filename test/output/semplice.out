ok
ok
ok
ok
ok 4
contenuto test
contenuto 
no
no
ok
ok
ok
ok /bar
ok /dir/bar
ok /foo/bar
ok
no
ok /foo
no
ok
contenuto 
ok
